kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowFragment.kt

```diff
@@ -61,22 +60,27 @@ abstract class WorkflowFragment<PropsT, OutputT : Any> : Fragment() {
   private lateinit var _runner: WorkflowRunner<OutputT>
 
   /**
-   * Provides subclasses with access to the products of the running [Workflow].
-   * Safe to call after [onActivityCreated].
+   * Provides the [ViewRegistry] used to display workflow renderings.
    */
-  protected val runner: WorkflowRunner<OutputT> get() = _runner
+  protected abstract val containerHints: ContainerHints
 
   /**
    * Called from [onActivityCreated], so it should be safe for implementations
    * to call [getActivity].
    */
   protected abstract fun onCreateWorkflow(): Config<PropsT, OutputT>
 
+  /**
+   * Provides subclasses with access to the products of the running [Workflow].
+   * Safe to call after [onActivityCreated].
+   */
+  protected val runner: WorkflowRunner<OutputT> get() = _runner
```

Comment:
Is this property actually being used outside of this class? I think it can be private.

Reply:
Intended to provide subclasses with access to the runner.

Reply:
Do they need access to it?